src/topology/vector_bundle/basic.lean

```diff
@@ -501,40 +446,69 @@ space) has a topological vector space structure with fiber `F` (denoted with
 which is linear in the fibers. -/
 class topological_vector_bundle :=
```

Comment:
I guess the state of affairs here is that this is only an intermediate refactor, and in the end you will have a base class with a trivialization atlas, and then `topological_vector_bundle` will extend this base class. Am I understanding correctly your plans?

Reply:
This is correct, although I was thinking the base class would contain both the trivialization atlas and the choice of a member at each basepoint.  See [here](https://github.com/leanprover-community/mathlib/blob/aa6649d7e235e46232e356d7cc81fd55bb7018a4/src/topology/new_fiber_bundle.lean#L430) in our `smooth-vector-bundle` branch.

Reply:
Yes, sure, the base class should contain both the atlas and the specific member at each basepoint. Perfect, then.